torch/_inductor/codecache.py

```diff
@@ -800,6 +811,10 @@ def _get_custom_pass_detail(
         return custom_pass.uuid()
 
 
+def has_frozen_params(gm: torch.fx.GraphModule) -> bool:
```

Comment:
Um, yuck. Is there a better way? This is brittle. I can set an attribute on `torch.fx.GraphModule` when freezing is applied?

Reply:
Yes - let's find a different solution. if you were to create a module with an attr `_frozen_param` this could fire. Attr set on the module when freezing is applied is fine with me.